### ISSUE TITLE
README installation steps are incomplete

### ISSUE THREAD

**Issue by adrianandreias** [danielwelch/hassio-zigbee2mqtt#339]
I find it pretty hard to follow your README installation steps.

I'm a developer and have general technical understanding of things, but I do not know home assistant.

Some steps give me no clue of where to actually perform them.

For instance:

* where do I actually find this UI widget: https://user-images.githubusercontent.com/7738048/40874668-bdd1645a-667a-11e8-88ff-03b78212910b.png
* and/or where do I put this yaml config after this image?

I have supervised Home Assistant installed.

Thanks

**Comment by groenmarsmannetje** [comment]
For the card/widget you need to add the yaml sections in the Home Assistant configuration files.

**Comment by danielwelch** [comment]
@adrianandreias the README here expects some basic understanding of home assistant. If you're unsure of the basics, like where to put configuration YAML or how to configure the UI, you should read the home assistant docs first (at the very least, the Configuration and Lovelace UI sections).

I do agree that the docs should be reworked, and I'd actually like to integrate them with zigbee2mqtt docs. That part of the docs is somewhat vague, as it was made before the era of Lovelace UI, and probably hasn't been updated since the visual UI updater became the "mainstream" way of doing things.
